core/src/navigation/LeftNav.html

```diff
@@ -231,24 +246,46 @@
     -webkit-box-shadow: 1px 1px 2px 0 rgba(0, 0, 0, 0.05);
     box-shadow: 1px 1px 2px 0 rgba(0, 0, 0, 0.05);
   }
-  .fd-side-nav__link {
-    .sap-icon--action {
-      //external-link icon
-      margin-left: 0.3em;
-      vertical-align: middle;
-    }
-    .fd-side-nav__icon {
-      width: 2em;
+
+  .fd-side-nav {
+    min-height: 100%;
+    margin-bottom: calc(-1 * #{$footerHeight});
+
+    &::after {
+      content: '';
+      display: block;
+      height: $footerHeight;
     }
-  }
-  .fd-side-nav__sublink {
-    &.fd-side-nav__link--icon {
-      padding-left: 16px;
+
+    &__link {
+      .sap-icon--action {
+        //external-link icon
+        margin-left: 0.3em;
+        vertical-align: middle;
+      }
+      .fd-side-nav__icon {
+        width: 2em;
+      }
     }
-    .fd-side-nav__icon {
-      width: 2em;
+
+    &__sublink {
+      &.fd-side-nav__link--icon {
+        padding-left: 16px;
+      }
+      .fd-side-nav__icon {
+        width: 2em;
+      }
     }
   }
+
+  .lui-sideNavFooter {
```

Comment:
```suggestion
  .lui-sideNavFooter {
     font-size: $side-nav-footer--font-size;
     line-height: $side-nav-footer--line-height;
```
Let's discuss it. I would go here with the solution to use our sizes (variables) and not rely on class from Fiori Fundamentals. If they change tomorrow `font-size` and `line-height` for class `fd-has-type-minus-1`, then our implementation will be broken. What do you think?

About `lui-sideNavFooter` name. I'm not sure about conventions to the naming. Do we use camel case css classes?

Reply:
I totally agree with the renaming, I'm doing it right now.

About defining everything apart from Fundamental - I think it might produce the opposite result from what you described. 95% of Luigi frontend relies on it. As I heard one day, _we should stick to Fundamental styles_. That's why I based the whole footer on it, doing as little custom styles as possible :)